apps/accounts/forms.py

```diff
@@ -66,6 +66,7 @@ def save(self):
                                    user_type="DEV",
                                    create_applications=True)
 
+        # TODO why do we do this (associate every user with a default patient)?
```

Comment:
I think this behavior needs to be removed.

https://github.com/CMSgov/bluebutton-web-server/pull/775